tests/migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -217,6 +217,7 @@ sub run {
         yast2_migration_handle_conflicts_text;
     }
 
+    assert_screen 'yast2-migration-summary', 120 if (get_var("SLE_PRODUCT", "hpc"));
```

Comment:
seems this assert should input in yast2_migration_handle_conflicts_text logic base current context.

Reply:
Updated code, it is right to move to yast2_migration_handle_conflicts_text and without hpc judgement since it is for textmode.
http://openqa.nue.suse.com/tests/3981379